.github/workflows/ci.yml

```diff
@@ -222,3 +222,53 @@ jobs:
         run: |
             export PATH=$HOME/bin:$PATH
             ./scripts/examples.sh
+
+  windows-build:
```

Comment:
Might be worth putting this in a different workflow file?

Reply:
windows.yml?

Reply:
but I'm a little hesitant to move this out, perhaps we can leverage all the geth, ganach stuff and unify everything in a matrix build with all vars etc, like I've attempted here https://github.com/gakonst/foundry/pull/218